### ISSUE TITLE
[BUG] Cam Test Fails on OAK-FFP-3P with 2x AR0234 and IMX378 

### ISSUE THREAD

**Issue by kneave** [luxonis/depthai-core#763]
**Check if issue already exists**

I have found no reference by searching online or the forums

**Describe the bug**
I have an OAK-FFP-3P with the following configuration:
Left: AR0234
Centre: IMX378
Right: AR0234

When I try to run cam_test.py using the develop branch of depthai-python (which includes depthai-core) it fails. The AR0234 are also not detected. Note, if I switch to the ar0234_x3 branch, all the cameras are detected.

Image showing only the IMX378 detected when using the `develop` branch:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/733411/225092363-2e594823-09f3-48ca-b4f6-c1efe1d953c7.png">

Image showing the same script reporting all three cameras, running from the `ar0234_x3` branch:
<img width="766" alt="image" src="https://user-images.githubusercontent.com/733411/225092661-21e4c76f-b569-464b-9dfb-c692cadf853d.png">

**Minimal Reproducible Example**
```
python cam_test.py -cams rgb,c left,c right,c            

Enabled cameras:
    rgb : color
   left : color
  right : color
DepthAI version: 2.20.2.0
DepthAI path: C:\Users\keegan\Documents\GitHub\depthai-python\venv\lib\site-packages\depthai.cp39-win_amd64.pyd
Traceback (most recent call last):
  File "C:\Users\keegan\Documents\GitHub\depthai-python\utilities\cam_test.py", line 181, in <module>
    cam[c].setIsp3aFps(args.isp3afps)
AttributeError: 'depthai.node.ColorCamera' object has no attribute 'setIsp3aFps'
```

**Expected behavior**
I would expect cam_test.py to succeed and be able to stream all cameras.

**Additional context**
I have attached the simple preview script I've been using to test the cameras on each branch.
[preview.zip](https://github.com/luxonis/depthai-core/files/10972042/preview.zip)

**Comment by szabi-luxonis** [comment]
When you switch to a branch, you also need to run `examples/install_requirements.py` to install the library specific to that branch.
You are using latest release, which is older than what's available on develop.

**Comment by kneave** [comment]
I ran `examples\install_requirements.py` before running the above test, I just deleted and grabbed a fresh copy from git and the same holds true:

<img width="708" alt="image" src="https://user-images.githubusercontent.com/733411/225100740-39695264-6769-4e95-88d8-eb610c715cba.png">

**Comment by szabi-luxonis** [comment]
If you download as `.zip` it won't work, since it relies on git versioning to retrieve firmware version.
You need to clone it, together with all its submodules.

**Comment by kneave** [comment]
For reference:

<img width="671" alt="image" src="https://user-images.githubusercontent.com/733411/225103199-ae2a0daa-73c0-4545-830f-61f6896e2add.png">

**Comment by szabi-luxonis** [comment]
What's the output of `python3 install_requirements.py` ?

**Comment by kneave** [comment]
Great minds, I was just grabbing that for you!

This is from `examples\install_requirements.py` <img width="965" alt="image" src="https://user-images.githubusercontent.com/733411/225104147-e8c26508-112d-41a1-bcf6-e278cd933ecc.png">

**Comment by szabi-luxonis** [comment]
Some of these commands fail:
https://github.com/luxonis/depthai-python/blob/develop/examples/install_requirements.py#L114
It's something related to your environment on windows.
Would you mind adding some debug prints and check? 
E.g.
```
    except (OSError, subprocess.CalledProcessError) as e:
        print(e)
        git_context = False
```

**Comment by kneave** [comment]
You're right, I'm on a new laptop and it turns out git isn't accessible in PyCharm for some reason as it was on my old one. I ran `install_requirements.py` from the PyCharm git-bash and it updated correctly though.

The isp error has gone, the AR0234 are still not detected however.

<img width="708" alt="image" src="https://user-images.githubusercontent.com/733411/225109745-733c671b-9053-4c05-9944-01f149d853c0.png">
